lib/reporter/global-reporter.js

```diff
@@ -51,7 +51,7 @@ module.exports = class GlobalReporter {
   setupChildProcessListener(emitter) {
     const Concurrency = require('../runner/concurrency/concurrency.js');
 
-    if (Concurrency.isMasterProcess()) {
+    if (Concurrency.isMasterProcess() || Concurrency.isChildProcess()) {
```

Comment:
shouldn't we remove the check altogether?

Reply:
Yes, we should, but i wasn't sure, why the check exist in the first place. Let me update the PR

Reply:
This check is for the scenario when running in parallel with test workers. A child process will send the results back to the main process. When running tests across multiple browsers, the results aren't called by the main process anymore so it doesn't work (since the bug), so we need to make sure that the test workers scenario still works well.

Reply:
we can probably just change the check to `!Concurrency.isTestWorker()`